src/tests/ftest/util/server_utils.py

```diff
@@ -41,8 +41,8 @@
 
 SESSIONS = {}
 
-DEFAULT_FILE = "src/tests/ftest/data/daos_server_baseline.yaml"
-AVOCADO_FILE = "src/tests/ftest/data/daos_avocado_test.yaml"
+DEFAULT_FILE = "lib/daos/TESTING/ftest/data/daos_server_baseline.yaml"
+AVOCADO_FILE = "lib/daos/TESTING/ftest/data/daos_avocado_test.yaml"
```

Comment:
This should migrate more towards where the server config file will be in a production environment -- probably $PREFIX/etc/daos, for example.  That will make it port to an RPM installation more easily.

Reply:
talking Phil's suggestion to use relative path.

Reply:
changed to $PREFIX/etc/daos